src/Microsoft.OData.Edm/Schema/EdmSingleton.cs

```diff
@@ -29,7 +29,7 @@ public EdmSingleton(IEdmEntityContainer container, string name, IEdmEntityType e
 
             this.container = container;
             this.entityType = entityType;
-            this.path = new EdmPathExpression(name);
+            this.path = new EdmPathExpression(this.container.FullName() + "/" + name);

```

Comment:
So we are changing the "Path" property on a singleton to be container-qualified.  Where is this path used?

In general, we should NOT be using container-qualified paths.

Reply:
revert

---
In reply to: [542982488](https://github.com/OData/odata.net/pull/1950#discussion_r542982488) [](ancestors = 542982488)